bundles/org.openhab.binding.melcloud/src/main/resources/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="melcloud" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
```

Comment:
The namespace has been changed:
```suggestion
	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
```
This comment applies to all places and xml files in the ESH-INF directory.